### ISSUE TITLE
usage of tx-notify beside get_transfer_by_txid

### ISSUE THREAD

**Issue by MedGaSToN** [monero-project/monero#8606]
Hello 
I run my own node and i use this on my own wallet
my problem is when trying to fetch a transaction info (pool|in|out) from my wallet using **get_transfer_by_txid** rpc method without adding the params **account_index** always the rpc call return transaction not found error. 
I know that using the params account_index will return the information but imagine looping throw 500+ account to fetch the result this cost a lot of time.
Q1 : how can i point directly the txid information without using account index?
Q2 : does tx-notify have an arg that tell the account index?
i realy take a lot of time to make it work but no success please i need help.
Thanks

**Comment by selsta** [comment]
Maybe using different accounts isn't the best for your use case? Why not use subaddresses instead?

**Comment by MedGaSToN** [comment]
yes i can use sub-addresses but in my case it the same logic.
I'm trying different logic to make the process more fast.
but it not possible to get the txid information without given the account index when meany account are present in the wallet ?
Thanks

**Comment by selsta** [comment]
> yes i can use sub-addresses but in my case it the same logic.

Can you rephrase?

> I'm trying different logic to make the process more fast.

Why would it be slower with subaddresses?

**Comment by MedGaSToN** [comment]
let say i have 5 account when a new transaction is detected (type pool or in) how can i get the txid information without knowing to witch account that txid is sent ?
